src/main/java/uk/gov/digital/ho/hocs/info/api/MemberService.java

```diff
@@ -42,28 +46,50 @@ Member getMember(UUID uuid) {
         }
     }
 
-    void updateWebMemberLists() {
+    public Integer updateWebMemberLists() {
         log.info("Started Updating Members Lists");
-        updateMember(listConsumerService.createFromWelshAssemblyAPI());
-        updateMember(listConsumerService.createFromScottishParliamentAPI());
-        updateMember(listConsumerService.createCommonsFromUKParliamentAPI());
-        updateMember(listConsumerService.createLordsFromUKParliamentAPI());
-        updateMember(listConsumerService.createFromIrishAssemblyAPI());
+        Long start = System.currentTimeMillis();
+
+        // launch async calls to all members endpoints, threads taken from common pool.
+        CompletableFuture<Set<Member>> futureWelsh
```

Comment:
Really like this!

Reply:
Ha ha, I see, yes - I'm with you - thought you were having a pop at my lovely code.